tutorial/practicals/nemo/1_nemo_psyir/README.md

```diff
@@ -236,18 +258,25 @@ are [`Loop`](https://psyclone-ref.readthedocs.io/en/latest/_static/html/classpsy
    `Assignment` nodes:
 
    ```python
-   form psyclone.psyir.nodes import Assignment
+   from psyclone.psyir.nodes import Assignment
    assignments = sched.walk(Assignment)
    ```
 
-4. Use the `view()` method of one of these `Assignments` nodes to
+4. Use the `view()` method of one of these `Assignment` nodes to
    examine its children. Check that you are able to work out
    which Fortran assignment this corresponds to, e.g.:
 
    ```python
    print(assignments[0].view())
    ```
 
+   As mentioned earlier, the `debug_string` method can also be very
+   useful:
+
+   ```python
+   print(assignments[0].debug_string())
+   ```
+
 ## 4. Conclusion
 
 Congratulations, you have now completed this section of the tutorial.
```

Comment:
A link to the next tutorial would be useful here. (also in the other 2 following tutorial readmes)

Reply:
Done in all three.